src/libraries/System.Runtime/tests/System/Int32GenericMathTests.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using Xunit;
+
+namespace System.Tests
+{
+    public sealed class Int32GenericMathTests : GenericMathTests<int>
```

Comment:
Is there a way to write a text template file to generate tests? This is a huge PR, it's not easy to see how much code coverage we have here :/

Reply:
In answer to both yours and @stephentoub comment: https://github.com/dotnet/runtime/pull/54650#discussion_r662594592

I've done a lot of testing locally, but a lot of this was "ad-hoc" and "generated" to validate various scenarios. I'm still working on porting these tests to be xUnit based and that's the main reason they aren't in this PR. I had, ideally, wanted to get this in so it makes the preview 7 cutoff and to try and get the tests up early next week.